### DISCUSSION TITLE
PTZ features for Eagle Eye IV USB camera by Polycom

### DISCUSSION THREAD

**Discussion by ethereally-coded** [obsproject/obs-studio#8726] [General]
Is there a software or driver I can download that will allow me to use my Polycom Eagle Eye IV USB camera's PTZ features without having to have all the windows of OBS open or without having to open the actual video camera's properties setting menu and then go to configure the video and use the slider? i want to be abkle to control the PTZ settings directly and on demand rather than having to open its configuration settings to do it....

**Comment by Fenrirthviti** [comment]
We do not provide integrations for these devices, they are third party plugins. I would recommend asking the plugin author, or Polycom for solutions as this is not really related to OBS development.
